tests/test_semantic_scene.py

```diff
@@ -38,7 +38,6 @@ def test_semantic_scene(scene, make_cfg_settings):
     if not osp.exists(scene):
         pytest.skip("Skipping {}".format(scene))
 
-    make_cfg_settings = {k: v for k, v in make_cfg_settings.items()}
```

Comment:
Is this suppose to be a `copy()`?

Reply:
The `make_cfg_settings` passed to the test is already a copy.

https://github.com/facebookresearch/habitat-sim/blob/fed4be9bcae818dc5e76aaa548f33a43c5ff6734/tests/conftest.py#L25

```
def make_cfg_settings():
    import habitat_sim.utils.settings

    cfg = habitat_sim.utils.settings.default_sim_settings.copy()
    {...}
    return cfg
```

Reply:
The `make_cfg_settings` passed to the test is already a copy.

https://github.com/facebookresearch/habitat-sim/blob/fed4be9bcae818dc5e76aaa548f33a43c5ff6734/tests/conftest.py#L25

```
def make_cfg_settings():
    import habitat_sim.utils.settings

    cfg = habitat_sim.utils.settings.default_sim_settings.copy()
    {...}
    return cfg
```